driver-mct/main/prep_ocn_mod.F90

```diff
@@ -1446,7 +1446,7 @@ subroutine prep_ocn_calc_w2x_ox(timer)
     call t_drvstartf (trim(timer),barrier=mpicom_CPLID)
     do ewi = 1,num_inst_wav
        w2x_wx => component_get_c2x_cx(wav(ewi))
-       call seq_map_map(mapper_Sw2o, w2x_wx, w2x_ox(ewi), norm=.true.)
+       call seq_map_map(mapper_Sw2o, w2x_wx, w2x_ox(ewi),fldlist=seq_flds_w2x_states, norm=.true.)
```

Comment:
Should this change also be held back for the flux coupling PR? I'm seeing seg faults in `SMS_D_Ln3.ne30pg2_IcoswISC30E3r5_wQU225Icos30E3r5.WCYCL1850-WW3.chrysalis_intel` and this seems to be a likely culprit:
```
 45: forrtl: error (78): process killed (SIGTERM)
 45: Image              PC                Routine            Line        Source     
 45: libpnetcdf.so.3.0  000015554B93D62C  for__signal_handl     Unknown  Unknown
 45: libpthread-2.28.s  00001555483F4CF0  Unknown               Unknown  Unknown
 45: libmpi.so.40.30.6  00001555488AAFC1  ompi_request_defa     Unknown  Unknown
 45: libmpi.so.40.30.6  00001555488E1083  MPI_Waitany           Unknown  Unknown
 45: libmpi_mpifh.so.4  0000155548E7D6FD  pmpi_waitany          Unknown  Unknown
 45: e3sm.exe           0000000010D6AF8F  m_rearranger_mp_r        1269  m_Rearranger.F90
 45: e3sm.exe           00000000008AFA29  seq_map_mod_mp_se         345  seq_map_mod.F90
 45: e3sm.exe           00000000004AC719  component_mod_mp_         908  component_mod.F90
 45: e3sm.exe           00000000004800F6  cime_comp_mod_mp_        4597  cime_comp_mod.F90
 45: e3sm.exe           0000000000460917  cime_comp_mod_mp_        3114  cime_comp_mod.F90
 45: e3sm.exe           000000000048F7E5  MAIN__                    153  cime_driver.F90
 45: e3sm.exe           0000000000427BE2  Unknown               Unknown  Unknown
 45: libc-2.28.so       0000155548057D85  __libc_start_main     Unknown  Unknown
 45: e3sm.exe           0000000000427AEE  Unknown               Unknown  Unknown
 45: [chr-0497:408078:0:408078] Caught signal 11 (Segmentation fault: address not mapped to object at address (nil))
```

Reply:
ah yes - separating the states from fluxes should all go in the flux coupling pr - sorry